routers/web/repo/view.go

```diff
@@ -374,6 +374,11 @@ func renderFile(ctx *context.Context, entry *git.TreeEntry, treeLink, rawLink st
 	ctx.Data["FileName"] = blob.Name()
 	ctx.Data["RawFileLink"] = rawLink + "/" + util.PathEscapeSegments(ctx.Repo.TreePath)
 
+	if ctx.Repo.TreePath == ".editorconfig" {
+		_, editorconfigErr := ctx.Repo.GetEditorconfig()
```

Comment:
Hmm, this error will always be in English.
I don't think there is anything we can do about that?

Reply:
Besides running third party errors through AI translation, no, I see no way 😉

Reply:
Thought so.